### PR TITLE
Add very quick start

### PR DESCRIPTION
**Pull request by richiejp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,28 +2,29 @@
...
 
-For developers and contributors interested in diving deeper into Prem Operator, we've made available a comprehensive [**Developer Guide**](./docs/developer_guide.md), a detailed [**Contribution Guide**](./docs/deployment.md), and a step-by-step [**Deployment Guide**](./docs/deployment.md). These resources are designed to help you get started, contribute to the project, and deploy AI models with ease.
+3. Deploy the Hermes + Big-AGI example and forward the ports
+    ```
+    $ kubectl -f exmaples/big-agi.yaml
+    $ kubectl port --address localhost --port 8080 forward big-agi-service --port 3000
+    ```
 
-In summary, Prem Operator is an essential tool for developers and organizations looking to deploy AI models efficiently, flexibly, and scalably within Kubernetes environments. Its design philosophy prioritizes ease of use, performance optimization, and collaborative improvement, making it an ideal choice for cutting-edge AI deployments.
+If you browse to localhost:3000 you'll be able to beging chatting as soon as
```
Comment (swarnimarun) [inline README.md:24]:
```suggestion
If you browse to localhost:3000 you'll be able to begin chatting as soon as
```

File: README.md [hunk 2]
```diff
@@ -2,28 +2,29 @@
...
+LocalAI downloads the model.
```
Comment (swarnimarun) [inline README.md:25]:
```suggestion
LocalAI downloads the model. All queries after the first one, where the model gets downloaded should be much faster.
```

File: README.md [hunk 3]
```diff
@@ -2,28 +2,29 @@
 
 ### Overview
 
-Easy deploy AI models to Kubernetes using LocalAI, vLLM, DeepSpeed-MII, NVIDIA Triton and custom Docker images.
+Deploy AI models and apps to Kubernetes without hitting every pitfall.
```
Comment (swarnimarun) [inline README.md:5]:
```suggestion
Deploy AI models and apps to Kubernetes without hitting any pitfalls and feeling overwhelmed by the ginormous YAMLs.
```

File: README.md [hunk 4]
```diff
@@ -2,28 +2,29 @@
...
+### Quick Start
 
-- 🌐**Unified Deployment Framework:** Offers a comprehensive set of Kubernetes controllers and custom resource definitions (CRDs) tailored for AI model deployment, eliminating the complexity of managing diverse AI engines.
-- 🛠**Engine Versatility:** While it harmonizes common configuration aspects, Prem Operator respects the unique capabilities of each AI engine, such as vLLM or Triton, facilitating optimal deployment strategies based on engine strengths.
-- 📦**Custom Resource Ecosystem:** Includes specialized resources like AI Deployment Custom Resource, AIModelMap, and AutoNodeLabeler, each with its controller, simplifying the intricate process of AI model deployment and management within Kubernetes environments.
-- 🔄**Seamless Integration:** Designed to integrate smoothly with existing Kubernetes clusters, enhancing the deployment, scaling, and management of AI models without disrupting current operations.
+Deploy the operator and chat with Hermes in 2 or 3 steps. Note that you need a NVIDIA GPU with 8GB of RAM.
 
-### Why Use Prem Operator?
+1. Install the NVIDIA Operator if not present already
```
Comment (swarnimarun) [inline README.md:11]:
```suggestion
1. Install the NVIDIA Operator if not present already (not needed if the host already has the relevant drivers installed, and is manually managed)
```

Comment (swarnimarun) [reply README.md:11]: I am not sure if we should add all this information so a more succint version of this.

File: README.md (hunk 2 above)
Comment (richiejp) [reply README.md:25]: This is clear in the video (which I haven't converted to a GIF yet, but you can see here https://youtu.be/MN8WpMG9PKs)

I think the information here has to be really minimal.

File: README.md (hunk 3 above)
Comment (richiejp) [reply README.md:5]: I don't think we can claim they won't hit any pitfalls :-)

File: README.md (hunk 4 above)
Comment (richiejp) [reply README.md:11]: The operator is at least needed for node labeling unless they label the nodes themselves. It is possible to install the operator without it configuring the runtime and drivers.

I don't really want to get into whether it's possible to use an NVIDIA card without the operator. Especially not in the quickstart example(s).

**Comment by richiejp** [comment]
https://github.com/premAI-io/prem-operator/assets/988098/a4186274-d01a-4ec6-90d9-d881a620047f

**Pull request merged**